.gazelcfg.json

```diff
@@ -0,0 +1,15 @@
+{
+	"GoPrefix": "k8s.io/kubernetes",
+	"SrcDirs": [
+		"./pkg",
+		"./cmd",
+		"./third_party",
+		"./plugin",
+		"./test",
+		"./federation"
+	],
+	"SkippedPaths": [
+		"pkg\\/client\\/clientset_generated\\/release_1_(3|4)",
```

Comment:
Those where added because they didn't compile after we remove the alpha apps api group. I don't think we are using this any longer so I'll remove.